### PR TITLE
Auth: password resets now working with Laravel 5.2

### PR DESCRIPTION
**Pull request by rundef**

### CODE REVIEW

File: src/Auth/ResetsPasswords.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+     */
+    protected function resetPassword($user, $password)
+    {
+    	$passwordService = new PasswordService;
+
+        $user->user_pass = $passwordService->wp_hash_password($password);
+
+        $user->save();
+        
+        Auth::guard($this->getGuard())->login($user);
```
Comment (jgrossi) [inline src/Auth/ResetsPasswords.php]: Here you must use the `Auth` instance from Laravel, but if the user is not using Laravel? I think maybe here it's better to use composition, making this class independent of Laravel `Auth`. What do you think?

Comment (rundef) [inline src/Auth/ResetsPasswords.php]: Hmmm, [I just took this code from Laravel](https://github.com/laravel/framework/blob/cf01460ff629cb80dfff3831c127a0b31edb9964/src/Illuminate/Foundation/Auth/ResetsPasswords.php#L214-L221), I only changed how the password is encrypted ...
If someone is not using Laravel, there's no need for him to use this trait.

**Comment by jgrossi** [comment]
@rundef BTW Should the `Corcel\Password\PasswordService` methods names have `snake_case`? Did you follow some convention here or can it use PSR-2?

**Comment by rundef** [comment]
I just took the function names as they were in WordPress, but yeah, this could be changed to PSR-2.

**Comment by jgrossi** [comment]
Ok, thanks ;-)

**Pull request merged**